.travis.yml

```diff
@@ -1,11 +1,20 @@
+dist: bionic
 language: c
 
 # Setting sudo access to false will let Travis CI use containers
 # rather than VMs to run the tests. For more details see:
 # https://docs.travis-ci.com/user/reference/overview/
 sudo: false
 
+before_install:
+  - sudo apt-get install python3-pip ninja-build
+  - pip3 install --upgrade pip
+  - pip3 install --user meson
+  - meson --version
+
 script:
+  - meson setup build
+  - ninja -C build/
```

Comment:
Please use `builddir` as build directory. See https://github.com/mesonbuild/meson/issues/1560.

I prefer to use Meson to compile:
```sh
meson builddir
meson compile -C builddir
```

Reply:
I switched to `builddir`.
I agree with you that `meson compile` is better but it was only added in `Meson 0.54.0` which is too recent for `focal`.
I'd leave `ninja -C builddir` for now and switch when it's available in a distribution package.